docs/getting_started/tutorial.rst

```diff
@@ -707,6 +708,107 @@ something like this:
    :alt: A simple tag view
 
 
+Categories
+~~~~~~~~~~
+
+Let's add a category system to our blog. Unlike tags, where a page author can bring a tag into existence simply by using it on a page, our categories will be a fixed list, managed by the site owner through a separate area of the admin interface.
+
+First, we define a ``BlogCategory`` model. A category is not a page in its own right, and so we define it as a standard Django ``models.Model`` rather than inheriting from ``Page``. Wagtail introduces the concept of "snippets" for reusable pieces of content that need to be managed through the admin interface, but do not exist as part of the page tree themselves; a model can be registered as a snippet by adding the ``@register_snippet`` decorator. All the field types we've used so far on pages can be used on snippets too - here we'll give each category an icon image as well as a name. Add to ``blog/models.py``:
+
+
+.. code-block:: python
+
+    from wagtail.wagtailsnippets.models import register_snippet
+
+
+    @register_snippet
+    class BlogCategory(models.Model):
+        name = models.CharField(max_length=255)
+        icon = models.ForeignKey(
+            'wagtailimages.Image', null=True, blank=True,
+            on_delete=models.SET_NULL, related_name='+'
+        )
+
+        panels = [
+            FieldPanel('name'),
+            ImageChooserPanel('icon'),
+        ]
+
+        def __str__(self):
+            return self.name
+
+        class Meta:
+            verbose_name_plural = 'blog categories'
+
+
+Note that we are using ``panels`` rather than ``content_panels`` here - since snippets generally have no need for fields such as slug or publish date, the editing interface for them is not split into separate 'content' / 'promote' / 'settings' tabs as standard, and so there is no need to distinguish between 'content panels' and 'promote panels'.
```

Comment:
Happy to consider alternative formatting here. I think the sentence does belong here in the tutorial though - I've seen various people in the support channels get this wrong, and I think it's more likely to stick in people's minds if we explain the reason for the difference rather than essentially leaving it at "it's different because it is".